rn/Teacher/ios/Teacher/SpeedGrader/SubmissionGrader.swift

```diff
@@ -79,6 +81,7 @@ struct SubmissionGrader: View {
         ))
         self.handleRefresh = handleRefresh
         self.studentAnnotationViewModel = StudentAnnotationSubmissionViewerViewModel(submission: submission)
+        self.settings = AppEnvironment.shared.subscribe(GetUserSettings(userID: "self"))
```

Comment:
Shouldn't we refresh this setting? This will just read from core data but won't check for changes even if the cache had already expired.